zebra/interface.c

```diff
@@ -2084,7 +2084,6 @@ static void zebra_if_dplane_ifp_handling(struct zebra_dplane_ctx *ctx)
 						       rc_bitfield);
 
 			if (if_is_no_ptm_operative(ifp)) {
-				bool is_up = if_is_operative(ifp);
 
 				ifp->flags = flags;
```

Comment:
So I am reading this as that flags contains the new set of the interface but ifp->flags did not on line 2087( which you removed ).  Why not set the ifp->flags and then grab the is_up variable afterwords?